.travis.yml

```diff
@@ -76,13 +77,13 @@ matrix:
     env: BUILD_ONLY=1 ARCH="i386"
 
 install:
-        - sudo apt-get install linux-headers-`uname -r`
-        - if [ -z "${ARCH}" -o "${ARCH}" == "x86_64" ] ; then
-               echo "compilling netmap";
+        - if [ ${NETMAP} ] ; then
```

Comment:
better to compare it with 1. If somebody will set NETMAP=0 if statement will be true.

Reply:
It that case shell complains when NETMAP is not defined, and I don't want to define NETMAP=0 for most test cases.

Reply:
No, in bash  echo ${NOT_DEFINED} is simple nothing. Bash never complains that variable is not defined. But if someone will define NETMAP=0 then it will be if [ "0" ]; then, which is actually true statement. I think after some time we can forgot that we or define it or not refine. I suggest here or define it as NETMAP=y or compare it with 1.

Reply:
if [ ${NOT_DEFINED} -eq 1]

Results this warning:
line 5: [: -eq: unary operator expected

Anyway, I'll try to add NETMAP=0 as global and then override that.